src/applications/letters/components/ErrorableSelect.jsx

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import _ from 'lodash';
+import { uniqueId, isArray, isString } from 'lodash';
```

Comment:
^^ https://github.com/department-of-veterans-affairs/vets-website/pull/11673/files#r377317714